api/src/main/java/org/pmiops/workbench/api/UserAdminController.java

```diff
@@ -28,30 +30,34 @@
 @RestController
 public class UserAdminController implements UserAdminApiDelegate {
 
-  private final UserService userService;
-  private final ProfileService profileService;
+  private final AccessModuleService accessModuleService;
   private final ActionAuditQueryService actionAuditQueryService;
+  private final ProfileService profileService;
   private final Provider<DbUser> userProvider;
   private final Provider<WorkbenchConfig> workbenchConfigProvider;
   private final TaskQueueService taskQueueService;
+  private final UserService userService;
 
   public UserAdminController(
-      UserService userService,
-      ProfileService profileService,
+      AccessModuleService accessModuleService,
       ActionAuditQueryService actionAuditQueryService,
+      ProfileService profileService,
       Provider<DbUser> userProvider,
       Provider<WorkbenchConfig> workbenchConfigProvider,
-      TaskQueueService taskQueueService) {
-    this.userService = userService;
-    this.profileService = profileService;
+      TaskQueueService taskQueueService,
+      UserService userService) {
+    this.accessModuleService = accessModuleService;
     this.actionAuditQueryService = actionAuditQueryService;
+    this.profileService = profileService;
+    this.taskQueueService = taskQueueService;
     this.userProvider = userProvider;
+    this.userService = userService;
     this.workbenchConfigProvider = workbenchConfigProvider;
-    this.taskQueueService = taskQueueService;
   }
 
   @Override
   @AuthorityRequired({Authority.ACCESS_CONTROL_ADMIN})
+  @Deprecated // use updateAccountProperties()
```

Comment:
updateAccountProperties() does everything this endpoint does (plus other things).

It also allows bypassing multiple modules with a single call. bypassAccessRequirement() only changes a single module.